src/App.tsx

```diff
@@ -12,7 +13,9 @@ const App = (props: AppProps) => {
             <StoreProvider fnr={props.fnr}>
                 <Nokkelinfo />
 
-                <div className="main_grid"></div>
+                <div className="main_grid">
+                    <CvInnhold />
```

Comment:
Fjerne denne om dette ikke er klar for å vises for veileder :)

Reply:
Kun lagt inn, for å teste i dev nå